lib/binding_web/tree-sitter-web.d.ts

```diff
@@ -8,7 +8,7 @@ declare module "web-tree-sitter" {
     delete(): void;
     parse(
       input: string | Parser.Input,
-      oldTree?: Parser.Tree,
+      oldTree?: Parser.Tree | null,
```

Comment:
Is this entire file redundant now? Can it be generated from the typescript source?